go.mod

```diff
@@ -23,9 +23,12 @@ replace github.com/Microsoft/hcsshim => github.com/Microsoft/hcsshim v0.8.8-0.20
 // Client-go does not handle different versions of mergo due to some breaking changes - use the matching version
 replace github.com/imdario/mergo => github.com/imdario/mergo v0.3.5
 
+replace cloud.google.com/go/meshtelemetry/v1alpha1 v0.0.0 => ./pkg/test/framework/components/stackdriver/edges
```

Comment:
Why replace instead of just importing `./pkg/test/framework/components/stackdriver/edges`? I think this will break anyone that imports this package, unless go modules is smarter than I think

Reply:
this matches what was done elsewhere for the same package. i was hoping to retain consistency.

the tests (and testing components) import `cloud.google.com/go/meshtelemetry/v1alpha1`, so go modules appears to be doing the correct thing here. this proto should only be used in testing, so hopefully other imports of the files is not an issue.

thoughts?

Reply:
I think I am missing something. I don't see any existing instances of a replace used like this, nor do I see anything using `cloud.google.com/go/meshtelemetry/v1alpha1` on Istio. Why do we need to import this as `cloud.google.com/go/meshtelemetry/v1alpha1` when such a module doesn't exist? Rather than directly importing `./pkg/test/framework/components/stackdriver/edges`.

My concern is when another project imports istio/istio (or uses `go get` it will try to find `./pkg/test/framework/components/stackdriver/edges` in their directory which doesn't exist). Or alternatively it will try to find `cloud.google.com/go/meshtelemetry/v1alpha1` which also doesn't exist. Its possible this concern is not valid

Reply:
https://github.com/istio/proxy/blob/3aaff15a85a5c0cc5c23635d4b858eee001df836/test/envoye2e/stackdriver_plugin/stackdriver.go#L25 is another place this replacement is used.  Looks like I screwed up naming the module in the local go.mod for it, but that's how it is being defined.

I don't feel all that strongly about it at all. The idea was that (eventually) `cloud.google.com/go/meshtelemetry/v1alpha1` would exist. I'm fine moving to the local package if we want to wait for the package to actually exist somewhere first.

Reply:
ok. removed the module altogether.